docs/install/index.rst

```diff
@@ -9,4 +9,5 @@ As of right now, the only officialy supported OS is Ubuntu 16.04.
    :maxdepth: 1
 
    install
+.. make_rootfs.md
```

Comment:
```suggestion
   make_rootfs
```
The above is correct.
But I think it is better to add doc of making root file system into this part (https://docs.blue-oil.org/install/install.html#optional-create-your-own-linux-image-on-microsd-card)
Thus, I mean it is better to edit install.md instead of adding a new markdown file.

Reply:
Can I let the document not linked ?

I think we shouldn't show the document of how to update to newer Ubuntu(Bionic Beaver) because the procedure is slightly complex like below.

[How to Update]
Copying the Ubuntu (Xenial Xerus) image we provides is required at first.
After removing all data in second partition once and uncompress *.tgz file to the second partition. Then kernel modules should be restored again.

These operations might be messy and the way to update will be changed soon because we'll prepare whole OS image instead of rootfs. Then I want to provide the documents for that.

Reply:
You mean this document is not enough to create a new OS image for blueoil, don't' you?
And I understood it is too complex to show all steps to create their own OS image before we provide a new base image with Ubuntu 18.04.
If my understanding is correct, we don't need this document into `/docs` directory. It is better to set as README.md on `make_os` dir.
But after providing new OS image based Ubuntu 18.04, we should add docs how to create it. So please create an issue not to forget to write fully document about the OS image creation.